.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 local
-.env
\ No newline at end of file
+.env
+dist
```

Comment:
You don't have a bunch of editor-specific or non-applicable entries in this .gitignore so I don't think @austin-rausch will like it

(jk, love you Ranch)